src/datanode/src/region_server.rs

```diff
@@ -272,6 +277,24 @@ impl RegionServerInner {
             Output::Stream(stream) => Ok(stream),
         }
     }
+
+    async fn stop(&self) -> Result<()> {
+        let region_ids = self.region_map.iter().map(|x| *x.key()).collect::<Vec<_>>();
+        info!("Stopping region server with regions: {:?}", region_ids);
+        self.region_map.clear();
+
+        let engines = self.engines.write().unwrap().drain().collect::<Vec<_>>();
+        // The underlying log store will automatically stop when all regions have been removed.
```

Comment:
Currently only the mito engine has a log store. This comment might be a little misleading. Move it to `MitoEngine::stop()` looks better